### DISCUSSION TITLE
DS18x20 not working since v13 tasmota at esp32 binary

### DISCUSSION THREAD

**Discussion by mpuff** [arendst/Tasmota#19807] [Q&A]
with tasmota 12.5 the ds18x20 sensor is working:
<img width="392" alt="image" src="https://github.com/arendst/Tasmota/assets/13885599/89d24f7d-0ce5-4088-9a89-b4fa2e61deb3">
<img width="362" alt="image" src="https://github.com/arendst/Tasmota/assets/13885599/769aba1e-ddc0-417b-be7e-5a6d23a88c40">

but since tasmota v13 it is not working anymore with the esp32 binary:
<img width="355" alt="image" src="https://github.com/arendst/Tasmota/assets/13885599/3749e0a8-863a-4fbf-9f44-8d9143281bbd">

any changes done with this sensor driver at v13?
it is not saving the ds18x20 sensor at gpis 16 after saving the settings. it is always none after the reboot to save the settings.

![image](https://github.com/arendst/Tasmota/assets/13885599/4404801f-cd85-4d77-add6-8175862bd819)

**Comment by Jason2866** [comment]
Moving to discussions since no issue template provided.

**Comment by mpuff** [comment]
I changed from GPIO16 to GPIO17, did not see the info in the documentation.
now it is working with the latest tasmota version
